tests/src/test/scala/cats/tests/NonEmptyListTests.scala

```diff
@@ -12,6 +12,7 @@ import cats.data.{NonEmptyList, NonEmptyVector}
 import cats.laws.discipline.arbitrary._
 import cats.laws.discipline.{ComonadTests, NonEmptyTraverseTests, MonadTests, ReducibleTests, SemigroupKTests, SerializableTests}
 
+@deprecated("to be able to test deprecated methods", since = "1.0.0-RC1")
```

Comment:
Sorry I hate to drag on this but I wouldn't deprecate the whole suite for one test. I'd rather we duplicated the test suite class with just the `concat` test in it and deprecate that one.

Reply:
Done.